roles/foreman_installer_devel_scenario/tasks/main.yml

```diff
@@ -16,10 +16,11 @@
   with_items:
     - katello-devel.yaml
 
-- name: 'Copy answers file'
+- name: "Create devel scenario answers file if it doesn't already exist"
```

Comment:
This will result in some options to this role only affecting the first run of the role and then never again -- that is generally an unexpected situation for Ansible roles.

If we are going this route, I should think we need to go all the way and make it such that the dynamic values we do have (https://github.com/theforeman/forklift/blob/master/roles/foreman_installer_devel_scenario/templates/katello-devel-answers.yaml#L3-L8) get passed to the installer as parameters.

Reply:
Thanks, I pushed an additional commit which moves these dynamic values out of the answers file and into `foreman_installer_options_internal_use_only`, which is probably where they should have been all along. (What clown put dynamic values in the answers file? Oh right, it's me. I'm the clown.)

So, the behavior should now be consistent with what you expect from an ansible role. Updating the `foreman_installer_devel_scenario_user` or `foreman_installer_devel_scenario_group` will again properly result in the installer making the appropriate changes.

I also removed `generate` and `deploy` from the certs params in the answers file, because the values we were passing were already the defaults for that module, so I couldn't see any reason why those needed to be there. I can revert that if it's needed for any reason.

I've tested this in my own setup, which does include a non-default value for `foreman_installer_devel_scenario_user` and it's all working just fine for me.